pkg/hwevent/event_ce_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 The Cloud Native Events Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package hwevent_test
+
+import (
+	"encoding/json"
```

Comment:
got format imports

Reply:
I run `go fmt ./...` and nothing changed?

Reply:
That won't do go imports formatting. right-click on your editor  and click on gotools -> goimports file